parm/archive/gdas.yaml.j2

```diff
@@ -101,6 +101,7 @@ gdas:
                 {% endfor %}
         - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowstat.tgz"
         - "{{ COMIN_CONF | relpath(ROTDIR) }}/{{ head }}snowanlvar.yaml"
+        - "{{ COMIN_SNOW_ANLMON | relpath(ROTDIR) }}/{{ head }}snow_iodastat.tgz"
```

Comment:
One question on this.  Is there a flag that is used to determine if the snow monitor jobs run, or do they always run if the snow analysis runs?

Reply:
Thanks!  That works for me.